.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "amazon-chime-sdk-component-library-react"]
+	path = amazon-chime-sdk-component-library-react
+	url = git@github.com:aws/amazon-chime-sdk-component-library-react.git
```

Comment:
The indention here seems to be not correct, could you please verify it?

Reply:
This is auto-generated by git (i.e., I didn't touch it) :D